airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -713,6 +714,11 @@ def trigger_reentry(self, context: Context, event: dict[str, Any]) -> Any:
             if not self.pod:
                 raise PodNotFoundException("Could not find pod after resuming from deferral")
 
+            if self.callbacks:
+                self.callbacks.on_operator_resuming(
+                    pod=self.pod, event=event, client=self.client, mode=ExecutionMode.SYNC
+                )
+
             if self.get_logs:
                 last_log_time = event and event.get("last_log_time")
                 if last_log_time:
```

Comment:
I think we need an `else` condition here that does
`self.write_logs(pod)` to have similar experience with earlier `execute_complete`

Reply:
in that case user does not want periodic log and it execute only in terminal state, right?